target_chains/ethereum/contracts/contracts/pulse/PulseState.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+contract PulseState {
+    uint8 public constant NUM_REQUESTS = 32;
+    bytes1 public constant NUM_REQUESTS_MASK = 0x1f;
+    uint8 public constant MAX_PRICE_IDS = 10;
```

Comment:
Is this enough?

I take it the problem is that you end up with higher gas usage if this limit is increased?

Reply:
yup, 10 seems like a reasonable estimate per request

Reply:
document why 10 is chosen and the consequences of it. we need to update the IPulse to also say at most 10 feeds are supported.